src/sign.ts

```diff
@@ -50,13 +50,7 @@ export class Signer {
     const digest = crypto.hash(payload);
 
     // Create Rekor entry
-    const proposedEntry = rekor.toProposedHashedRekordEntry(
-      digest,
-      sigMaterial
-    );
-    const entry = await this.rekor.createEntry(proposedEntry);
-
-    return bundle.toMessageSignatureBundle(digest, sigMaterial, entry);
+    return this.tlog.createMessageSignatureEntry(digest, sigMaterial);
```

Comment:
All of the data transformation necessary to post to Rekor and format the response as a Bundle is now encapsulated behind the `TLog` interface.